test/python/visualization/test_pass_manager_drawer.py

```diff
@@ -62,13 +62,43 @@ def test_pass_manager_drawer_style(self):
                  EnlargeWithAncilla: 'pink',
                  RemoveResetInZeroState: 'grey'}
 
-        filename = self._get_resource_path('current_l1.png')
-        level_1_pass_manager(self.config).draw(filename=filename, style=style)
+        filename = self._get_resource_path('current_l1.dot')
+        level_1_pass_manager(self.config).draw(filename=filename, style=style, raw=True)
 
-        self.assertImagesAreEqual(filename,
-                                  path_to_diagram_reference('pass_manager_l1_style_ref.png'))
+        self.assertDotFilesAreEqual(filename,
+                                    path_to_diagram_reference('pass_manager_style_l1.dot'))
         os.remove(filename)
 
+    def assertDotFilesAreEqual(self, current, expected):
+        """
+        Asserts that 2 dot files are the same - can't use a straightforward
+        line by line comparison as clusters are named randomly by pydot and
+        so the files will always differ as the cluster names will be different
+        """
```

Comment:
Could the cluster name be made deterministic if we removed the `str(id(controller_group))` from https://github.com/Qiskit/qiskit-terra/blob/256f719/qiskit/visualization/pass_manager_visualization.py#L90 ?